### PR TITLE
Added glowstone, glowstone dust, and glowstone dust drop percentages.

### PR DESCRIPTION
**Pull request by ImDaRealAni**
Added glowstone, glowstone dust, and glowstone dust drop percentages.

### CODE REVIEW

File: dragonfly/item/glowstone_dust.go [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+type GlowstoneDust struct {}
+
+func (g GlowstoneDust) EncodeItem() (id int32, meta int16) {
+	return 348, 0
+}
+
+func (g GlowstoneDust) MaxCount() int {
+	return 64
+}
```
Comment (TwistedAsylumMC) [inline dragonfly/item/glowstone_dust.go:11]: This can be removed

File: main.go [hunk 2]
```diff
@@ -36,6 +37,9 @@ func main() {
 		log.Fatalln(err)
 	}
 
+	w := server.World()
+	w.SetDefaultGameMode(gamemode.Creative{})
+
```
Comment (TwistedAsylumMC) [inline main.go:42]: Please remove this test code

File: dragonfly/block/glowstone.go [hunk 3]
```diff
@@ -0,0 +1,38 @@
+package block
+
+import (
+	"github.com/df-mc/dragonfly/dragonfly/item"
+	"math/rand"
+)
+
+// Glowstone is commonly found on the ceiling of the nether dimension.
+type Glowstone struct{ noNBT }
```
Comment (TwistedAsylumMC) [inline dragonfly/block/glowstone.go:9]: Pretty sure there must be a space after struct

File: dragonfly/block/glowstone.go [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+// EncodeBlock ...
+func (g Glowstone) EncodeBlock() (name string, properties map[string]interface{}) {
+	return "minecraft:glowstone", nil
+}
+
+// LightEmissionLevel returns 15.
+func (Glowstone) LightEmissionLevel() uint8 {
+	return 15
+}
+
```
Comment (TwistedAsylumMC) [inline dragonfly/block/glowstone.go:35]: Add // Hash ...

File: dragonfly/item/glowstone_dust.go [hunk 5]
```diff
@@ -0,0 +1,11 @@
+package item
+
+type GlowstoneDust struct {}
+
+func (g GlowstoneDust) EncodeItem() (id int32, meta int16) {
```
Comment (TwistedAsylumMC) [inline dragonfly/item/glowstone_dust.go:5]: Add // EncodeItem ...

File: dragonfly/item/glowstone_dust.go [hunk 6]
```diff
@@ -0,0 +1,11 @@
+package item
+
+type GlowstoneDust struct {}
```
Comment (TwistedAsylumMC) [inline dragonfly/item/glowstone_dust.go:3]: Please document this struct

File: dragonfly/block/glowstone.go (hunk 3 above)
Comment (Sandertv) [reply dragonfly/block/glowstone.go:9]: For single line structs I think this is how Go formats it.

**Comment by ImDaRealAni** [comment]
Will add all the documentation you guys requested.
Thanks.

File: dragonfly/item/glowstone_dust.go [hunk 7]
```diff
@@ -1,11 +1,9 @@
 package item
 
+// Glowstone Dust is dropped when breaking the glowstone block .
```
Comment (T14Raptor) [inline dragonfly/item/glowstone_dust.go:3]: "GlowstoneDust" not "Glowstone Dust"

File: dragonfly/item/glowstone_dust.go [hunk 8]
```diff
@@ -0,0 +1,9 @@
+package item
+
+// GlowstoneDust is dropped when breaking the glowstone block .
```
Comment (Sandertv) [inline dragonfly/item/glowstone_dust.go:3]: Please remove the period between the `k .`

**Pull request merged**